src/container/login/Login.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
This file seems to be redundant. Consider removing it.

Reply:
good observation. I have removed it.